packages/driver/src/multi-domain/index.ts

```diff
@@ -13,6 +14,11 @@ import { SpecBridgeDomainCommunicator } from './communicator'
 
 const specBridgeCommunicator = new SpecBridgeDomainCommunicator()
 
+const webSocket = client({
```

Comment:
I am wondering if we need to disconnect this at some point to prevent a buildup of socket connections. Should we `connect` during the `setup()` and then maybe `disconnect` when either `done` or `queue:finished` is invoked?

Reply:
This might also not be in scope here too, which is ok!

Reply:
I don't think we should disconnect in the cases of `done` or `queue:finished`. We may need to unbind listeners when we re-setup on test re-run, but that's not in place yet, so we can hold off for now and worry about it when that's implemented.

Reply:
IMO cleanup should be in scope.

Reply:
I don't think there is anything to cleanup yet until re-setup is implemented.